core/modules/installer/css/installer.css

```diff
@@ -0,0 +1,251 @@
+#installer-browser-install-button-form {
+  clear: both;
+  margin: 10px 0 0;
+}
+
+#installer-browser-install-queue {
+  margin: 5px 0 0;
+}
+
+.installer-browser-install-queue-item {
+  margin: 5px 0;
+}
+
+.installer-browser-install-queue-items {
+  clear: both;
+  margin: 0 0 15px;
+}
+
+.installer-browser-install-link {
+  clear: both;
+}
+
+.installer-browser-install-queue-item a {
+  padding-left: 20px;
+  background: url("../images/red-x.png") no-repeat scroll left 2px transparent;
+}
+
+.installer-browser-selected-release {
+  display: none;
+}
+
+.installer-browser-show-releases-link {
+  cursor: pointer;
+}
+
+div.item-list ul.installer-browser-sort-list, 
+div.item-list ul.installer-browser-servers-list {
+  margin: 0;
+}
+
+div.item-list ul.installer-browser-sort-list li, 
+div.item-list ul.installer-browser-servers-list li {
```

Comment:
Some specificity wars happening here actually. Is it possible to remove `div.item-list ul` and just have this be `.installer-browser-sort-list li`?